features/running-tests.feature

```diff
@@ -7,3 +7,7 @@ Feature: Running tests
         Given this step exists
         When I run "python manage.py behave"
         Then I should see the behave tests run
+
+    Scenario: Test django_ready
+        When I run "python manage.py behave"
+        Then context.django should be declared
```

Comment:
I think the `Then` shouldn't refer to the test implementation. Maybe `Then a running Django instance is available in django_ready`, or so? (Oh, horribly long sentence!)

Reply:
Hmmm, not sure if this `Then` should just test that `django_ready` is called, or that Django is actually _ready_. :thinking: